omod/src/main/webapp/pages/login.gsp

```diff
@@ -19,67 +19,74 @@
 
 ${ ui.includeFragment("referenceapplication", "infoAndErrorMessages") }
 
+<% if(showSessionLocations) { %>
 <script type="text/javascript">
     jQuery(function() {
-    	updateSelectedOption = function() {
-	        jQuery('#sessionLocation li').removeClass('selected');
-	        
-			var sessionLocationVal = jQuery('#sessionLocationInput').val();
-	        if(sessionLocationVal != null && sessionLocationVal != "" && sessionLocationVal != 0){
-	            jQuery('#sessionLocation li[value|=' + sessionLocationVal + ']').addClass('selected');
-	        }
-    	};
-    
+        updateSelectedOption = function() {
```

Comment:
This indentation should be fixed since wrong in the original file - https://github.com/openmrs/openmrs-module-referenceapplication/blob/master/omod/src/main/webapp/pages/login.gsp#L25

Reply:
If you are fixing indention, then i would recommend a separate pull request for it. That way, it becomes easier for me to tell the difference between indention changes and those require to implement this functionality.

Reply:
So then, I will remove those from here as well